tests.py

```diff
@@ -100,3 +89,13 @@ def test_sip_api_qtpy():
         assert sip.getapi("QString") == 2, ("PyQt4 API version should be 2, "
                                             "instead is %s"
                                             % sip.getapi("QString"))
+
+
+def test_sip_api_pyqt4():
+    """PyQt4 should have sip version 1"""
```

Comment:
What exactly are you testing here? Seem to be testing PyQt4 rather than Qt.py.

Perhaps test for something like this?

```python
import Qt
assert_raises(ImportError, __import__, "PyQt4")
```